### PR TITLE
Add minutes from editorial sessions

### PR DESCRIPTION
**Pull request by justinwb**
Per the agreement by editors in the editorial session on 8/31, creating a place to store minutes  from editorial sessions moving forward, and adding minutes from previous sessions.

*Note: Some minutes from older sessions (especially from previous years) may not be included. Editors - please add if you have them.*

### CODE REVIEW

**Review by kjetilk — APPROVED** [review]
I'll trust you on this, @justinwb , since you've been very diligent getting the hackmd up for every session!

File: meetings/2020-09-04.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Solid Editorial Session
+September 4th, 2020
```
Comment (csarven) [inline meetings/2020-09-04.md:2]:
```suggestion
Date: 2020-09-04
```

File: meetings/2020-11-19.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Solid Editorial Session
+2020-11-19
```
Comment (csarven) [inline meetings/2020-11-19.md:2]:
```suggestion
Date: 2020-11-19
```

File: meetings/2020-11-30.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Solid Editorial Session
+November 30th, 2020
```
Comment (csarven) [inline meetings/2020-11-30.md:2]:
```suggestion
Date: 2020-11-30
```

File: meetings/2021-01-04.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# Solid Editorial Session
+January 4th, 2021
```
Comment (csarven) [inline meetings/2021-01-04.md:2]:
```suggestion
Date: 2021-09-04

```

File: meetings/2021-02-16.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
+# Solid Editorial Meeting
+February 16th, 2021
```
Comment (csarven) [inline meetings/2021-02-16.md:2]:
```suggestion
Date: 2021-02-16
```

File: meetings/2021-03-02.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# Solid Editorial Meeting
+March 2nd, 2021
```
Comment (csarven) [inline meetings/2021-03-02.md:2]:
```suggestion
Date: 2021-03-02
```

File: meetings/2021-04-06.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
+# Solid Editorial Meeting
+April 6th, 2021
```
Comment (csarven) [inline meetings/2021-04-06.md:2]:
```suggestion
Date: 2021-04-06
```

File: meetings/2021-04-22.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
+# Solid Editorial Session - Specification Features
+
+Date: 2021-04-22
```
Comment (csarven) [inline meetings/2021-04-22.md:3]:
```suggestion
Date: 2021-04-22
```

File: meetings/2021-05-04.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
+# Solid Editorial Session
+May 4th, 2021
```
Comment (csarven) [inline meetings/2021-05-04.md:2]:
```suggestion
Date: 2021-05-04
```

File: meetings/2021-06-01.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
+# Solid Editorial Session
+June 1st, 2021
```
Comment (csarven) [inline meetings/2021-06-01.md:2]:
```suggestion
Date: 2021-06-01
```

File: meetings/2021-06-15.md [hunk 11]
```diff
@@ -0,0 +1,88 @@
+# Solid Weekly Editorial Meeting
+June 15th, 2021
```
Comment (csarven) [inline meetings/2021-06-15.md:2]:
```suggestion
Date: 2021-06-15
```

File: meetings/2021-06-22.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
+# Weekly Editorial Discussion
+
+DateTime: 2021-06-22T12:00:00Z
```
Comment (csarven) [inline meetings/2021-06-22.md:3]:
```suggestion
Date: 2021-06-22

```

File: meetings/2021-06-29.md [hunk 13]
```diff
@@ -0,0 +1,70 @@
+# Solid Weekly Editorial
+June 29th, 2021
```
Comment (csarven) [inline meetings/2021-06-29.md:2]:
```suggestion
Date: 2021-06-29
```

File: meetings/2021-07-06.md [hunk 14]
```diff
@@ -0,0 +1,47 @@
+# Solid Weekly Editorial
+July 6th, 2021
```
Comment (csarven) [inline meetings/2021-07-06.md:2]:
```suggestion
Date: 2021-07-06
```

File: meetings/2021-07-20.md [hunk 15]
```diff
@@ -0,0 +1,43 @@
+# Solid Weekly Editorial
+
```
Comment (csarven) [inline meetings/2021-07-20.md:2]:
```suggestion
Date: 2021-07-20

```

File: meetings/2021-08-03.md [hunk 16]
```diff
@@ -0,0 +1,21 @@
+# Solid Weekly Editorial
+
```
Comment (csarven) [inline meetings/2021-08-03.md:2]:
```suggestion
Date: 2021-08-03

```

File: meetings/2021-08-10.md [hunk 17]
```diff
@@ -0,0 +1,48 @@
+# Solid Weekly Editorial
+
```
Comment (csarven) [inline meetings/2021-08-10.md:2]:
```suggestion
Date: 2021-08-10

```

File: meetings/2021-08-31.md [hunk 18]
```diff
@@ -0,0 +1,89 @@
+# Solid Editorial Session
+August 31st, 2021
```
Comment (csarven) [inline meetings/2021-08-31.md:2]:
```suggestion
2021-08-31
```

**Review by csarven — CHANGES_REQUESTED** [review]
There were a few missing dates in the content. Better to have those in so that if/when the files are relocated (losing {date}.md), the content will retain the date.

**Review by csarven — APPROVED** [review]

**Comment by csarven** [comment]
Oh.. forgot to mention: the minutes should not be available under solidproject.org/TR/ - that's reserved for technical reports. I don't know where else would be more suitable - perhaps for the time being, it only needs to be available from GitHub.

**Comment by kjetilk** [comment]
> Oh.. forgot to mention: the minutes should not be available under solidproject.org/TR/ - that's reserved for technical reports. I don't know where else would be more suitable - perhaps for the time being, it only needs to be available from GitHub.

The main idea was to have them on a pod, right?

**Comment by csarven** [comment]
Yes, I just meant the /path/to/minutes/

Perhaps we use something like `/minutes/{team, panel, ..}/`

**Review by kjetilk — APPROVED** [review]

**Pull request merged**